Mathlib/Data/Fintype/List.lean

```diff
@@ -16,47 +17,92 @@ admits a `Fintype` instance.
 
 ## Implementation details
 To construct the `Fintype` instance, a function lifting a `Multiset α`
-to the `Finset (List α)` that can construct it is provided.
+to the `Multiset (List α)` that can construct it is provided.
```

Comment:
"that can construct it"? Not sure what this is meant to mean.

Reply:
done, removed that part